environment/src/main/java/com/sequenceiq/environment/parameters/validation/validators/parameter/AwsParameterValidator.java

```diff
@@ -24,15 +27,17 @@
 
     private final ParametersService parametersService;
 
-    public AwsParameterProcessor(NoSqlTableCreationModeDeterminerService noSqlTableCreationModeDeterminerService,
+    public AwsParameterValidator(NoSqlTableCreationModeDeterminerService noSqlTableCreationModeDeterminerService,
             ParametersService parametersService) {
         this.noSqlTableCreationModeDeterminerService = noSqlTableCreationModeDeterminerService;
         this.parametersService = parametersService;
     }
 
-    public ValidationResult processAwsParameters(EnvironmentDto environmentDto, ParametersDto parametersDto) {
-        ValidationResultBuilder validationResultBuilder = new ValidationResultBuilder();
+    public ValidationResult validate(EnvironmentDto environmentDto, ParametersDto parametersDto, ValidationResultBuilder validationResultBuilder) {
         AwsParametersDto awsParametersDto = parametersDto.getAwsParametersDto();
+        if (Objects.isNull(awsParametersDto)) {
+            return validationResultBuilder.build();
```

Comment:
We should use ValidationResult.empty() instead

Reply:
Please see above